src/font.rs

```diff
@@ -77,7 +77,7 @@ fn is_zero(v: &u32) -> bool {
 impl Default for MetaInfo {
     fn default() -> Self {
         MetaInfo {
-            creator: DEFAULT_METAINFO_CREATOR.to_string(),
+            creator: Some(DEFAULT_METAINFO_CREATOR.to_string()),
```

Comment:
Maybe just default to `None`?

Reply:
Hm, and set the creator on saving? I'd prefer to keep the explicit norad reference, since norad is the only user of it currently.